src/components/AboutPage/TeamSection.js

```diff
@@ -2,232 +2,73 @@ import React from "react";
-            width: 197,
-            height: 224,
-            quality: 70,
-            placeholder: BLURRED,
-            blurredOptions: {width: 100}
-          )
-        }
-      }
-      JesdakornSamittiauttakorn: file(
-        relativePath: {eq: "team/Jesdakorn Samittiauttakorn.jpeg"}
-      ) {
-        childImageSharp {
-          gatsbyImageData(
-            width: 197,
-            height: 224,
-            quality: 70,
-            placeholder: BLURRED,
-            blurredOptions: {width: 100}
-          )
-        }
-      }
-      ArsalanNi: file(
-        relativePath: {eq: "team/Arsalan Ni.jpeg"}
-      ) {
-        childImageSharp {
-          gatsbyImageData(
-            width: 197,
-            height: 224,
-            quality: 70,
-            placeholder: BLURRED,
-            blurredOptions: {width: 100}
-          )
-        }
-      }
-      LenaNamiki: file(
-        relativePath: {eq: "team/Lena Namiki.jpeg"}
-      ) {
-        childImageSharp {
-          gatsbyImageData(
-            width: 197,
-            height: 224,
-            quality: 70,
-            placeholder: BLURRED,
-            blurredOptions: {width: 100}
-          )
+      WordpressData: wpcontent {
+        teamMembers {
+          nodes {
+            featuredImage {
+              node {
+                sourceUrl
+                imageFile {
+                  childImageSharp {
+                    gatsbyImageData(
+                      width: 197
+                      height: 224
+                      quality: 70
+                      placeholder: BLURRED
+                      blurredOptions: {width: 100}
+                    )
+                  }
+                }
+              }
+            }
+            teamMemberData {
+              bio1
+              bio2
+              bio3
+              name
+              title
+            }
+            teamMemberDataJa {
+              bio1
+              bio2
+              bio3
+              name
+              title
+            }
+          }
         }
       }
     }
   `)
 
-  const { SeiraYun, AtsushiInoue, GalinaNeydenova, RhysMalyon, LawrenceCate, JaysonIlagan, RiteshGiri, XanVarcoe, TinCabrera, PranavKhanna, CristhianTejada, RaphaelHode, JesdakornSamittiauttakorn, ArsalanNi, LenaNamiki } = data
-
-  const teamImages = [SeiraYun, AtsushiInoue, GalinaNeydenova, RhysMalyon, LawrenceCate, JaysonIlagan, RiteshGiri, XanVarcoe, TinCabrera, PranavKhanna, CristhianTejada, RaphaelHode, JesdakornSamittiauttakorn, ArsalanNi, LenaNamiki]
+  const { WordpressData } = data
 
   const { language } = props.pageContext
 
   const meetTheTeamLanguage = language === 'ja' ? 'チーム' : 'Meet The Team'
-  const teamMemberComponentLanguageSelector = language === 'ja' ? teamMembersJa : teamMembers
+  const teamMemberData = language === 'ja' ? "teamMemberDataJa" : "teamMemberData"
+
+  const team = WordpressData.teamMembers.nodes.map((member, index) => {
+      if(!member.featuredImage)
```

Comment:
@pragyachik this means that when team members don't have image their profile will not be rendered. Can we check a much more stable field instead like `name or id`.

Reply:
I had added the check as it was possible for the featuredImage field to be null. I have now removed it in my recent commit, and made it a mandatory field in WordPress. I am not sure if I should add checks for individual fields like name/featuredImage as it might be excessive, so I haven't added them yet.